test/crv-staking/IncentivisedVotingLockup.ts

```diff
@@ -821,6 +827,79 @@ describe("IncentivisedVotingLockup", () => {
       expect(redeemerRBNBalanceAfter).eq(
         redeemerRBNBalanceBefore.add(amountToSeize)
       );
+
+      await votingLockup
+        .connect(david.signer)
+        .createLock(stakeAmt1, (await getTimestamp()).add(ONE_WEEK.add(1)));
+      const davidData = await snapshotData(david);
+      // amount.mul(totalShares).div(totalRBN);
+      let newDavidShares = stakeAmt1
+        .mul(stakeAmt1.add(stakeAmt2))
+        .div(stakeAmt1.add(stakeAmt2).sub(amountToSeize));
+
+      expect(davidData.userLocked.shares).eq(newDavidShares);
+      expect(davidData.userLocked.amount).eq(stakeAmt1);
+
+      await votingLockup
+        .connect(charlie.signer)
+        .createLock(stakeAmt2, (await getTimestamp()).add(ONE_WEEK.add(1)));
+      const charlieData = await snapshotData(charlie);
+      // amount.mul(totalShares).div(totalRBN);
+      let newCharlieShares = stakeAmt2
+        .mul(stakeAmt1.add(stakeAmt2).add(newDavidShares))
+        .div(stakeAmt1.mul(2).add(stakeAmt2).sub(amountToSeize));
+
+      expect(charlieData.userLocked.shares).eq(newCharlieShares);
+      expect(charlieData.userLocked.amount).eq(stakeAmt2);
+
+      await increaseTime(ONE_WEEK.mul(2));
+
+      // alice withdraws
+      const aliceBefore = await snapshotData(alice);
+      let aliceToWithdraw = aliceBefore.userLocked.shares
+        .mul(aliceBefore.totalLocked)
+        .div(await votingLockup.totalShares());
+
+      await votingLockup.connect(alice.signer).withdraw();
+      const aliceAfter = await snapshotData(alice);
+
+      // depositAmount / 2 < alice withdraw Amount < depositAmount * 2 / 3
+      // she deposited 10 rbn into total of 1010 rbn, and we withdrew 500 => 10 * (1010 - 500) / 1010
+
+      expect(aliceToWithdraw).lt(aliceBefore.userLocked.amount.mul(2).div(3));
+
+      expect(aliceToWithdraw).gt(aliceBefore.userLocked.amount.div(2));
+
+      expect(aliceToWithdraw).eq(
+        stakeAmt1
+          .mul(stakeAmt1.add(stakeAmt2).sub(amountToSeize))
+          .div(stakeAmt1.add(stakeAmt2))
+      );
+
+      expect(aliceAfter.senderStakingTokenBalance).eq(
+        aliceBefore.senderStakingTokenBalance.add(aliceToWithdraw)
+      );
+
+      // david withdraws
+      const davidBefore = await snapshotData(david);
+      let davidToWithdraw = davidBefore.userLocked.shares
+        .mul(davidBefore.totalLocked)
+        .div(await votingLockup.totalShares());
+
+      await votingLockup.connect(david.signer).withdraw();
+      const davidAfter = await snapshotData(david);
+
+      // depositAmount = bob withdraw amount
+      // he deposited rbn after seize and so he received more shares
+      // which equals same deposit amount
+
+      // there is rounding error
+
+      expect(davidBefore.userLocked.amount).eq(davidToWithdraw.add(1));
```

Comment:
there is a rounding error by 1

ex: 

```
deposit 100 * 10 ** 18
(no redeems before withdraw)
withdraw 100 * 10 ** 18 - 1
```

but no rounding error when there is redeem before withdraw.